doc/rbd/rbd-persistent-cache.rst

```diff
@@ -38,10 +38,10 @@ Enable RBD Shared Read-only Parent Image Cache
 ----------------------------------------------
 
 To enable RBD shared read-only parent image cache, the following Ceph settings
-need to added in the ``[client]`` `section`_ of your ``ceph.conf`` file.
-
-``rbd parent cache enabled = true``
+need to added in the ``[client]`` `section`_ of your ``ceph.conf`` file::
 
+        rbd parent cache enabled = true
+        rbd plugins = parent_cache
```

Comment:
Shouldn't we add this to the pending release notes? Because if a user already had `rbd parent cache enabled` in it config, it might not notice it would not work after upgrade.

Alternatively we could add parent_cache to the default value for `rbd plugins`, though I am not sure I like this idea. I like the librbd loads as little as possible. Probably it does not make much difference though...

Reply:
Thanks. Yes, I should add it to the `PendingReleaseNotes`.